src/main/java/com/jvm_bloggers/frontend/public_area/blogs/PersonalBlogsPage.java

```diff
@@ -0,0 +1,24 @@
+package com.jvm_bloggers.frontend.public_area.blogs;
+
+import com.jvm_bloggers.entities.blog.BlogType;
+
+import org.wicketstuff.annotation.mount.MountPath;
+
+@MountPath(value = "blogs", alt = "blogs/personal")
```

Comment:
Generally I would mount these three pages in a following way:
`/personal-blogs`
`/company-blogs`
`/video-channels`